apps/portal/src/app/shared/services/file.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { Apollo, gql } from 'apollo-angular';
+import { GraphqlService } from '../../graphql/graphql.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FileService {
+  constructor(
+    private graphqlService: GraphqlService,
+    private apollo: Apollo,
+  ) {}
+
+  fetchSummaries(jobIds: number[]) {
+    const jobIdsString = JSON.stringify(jobIds);
+
+    const query = gql`
```

Comment:
Whyare these added here?
this should be in querries file.
And should be using graphQL service to call mutate or query like in osmo-x or assistant app.

Reply:
yes fixed that